d3servercheck.ps1

```diff
@@ -1,70 +1,52 @@
-function Write-Server-Result($tcpconns) {
+function Write-ServerResult($tcpconns) {
     if (!$tcpconns) {
         return
     }
-
-    foreach ($tcpconn in $tcpconns) {
-        Write-Host "$($tcpconn.RemoteAddress) (" -NoNewline
-        if ($is_good_ip.ContainsKey($tcpconn.RemoteAddress)) {
-            if ($is_good_ip[$tcpconn.RemoteAddress]) {
-                Write-Host "GOOD" -NoNewline
+    $Unique_Addresses = $tcpconns | Sort-object -property RemoteAddress -unique
+    write-host "Connections:"
```

Comment:
This messes up the formatting as it was originally, and I would like to keep the original version: One line per check detailing the server ips. Otherwise the output becomes too difficult to parse on a quick glance, as I understand it this script writes at least two lines every iteration, leading to a lot of vertical movement.